fix(user migration): clarifying scim steps (#6654)
newrelic/docs-website: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

### Managing users [#managing-users]

For how users are added and managed, you can select **Manually** or **Identity provider**. The option to use your identity provider to provision users is available only if your organization has [Pro or Enterprise edition](https://newrelic.com/pricing). 

> If you choose **Identity provider**, **you must follow the [steps for automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign#how-to)**. Once you completely those steps return to the user migration wizard and docs. 

Reviewer: Once you complete* 
```suggestion
If you choose **Identity provider**, **you must follow the [steps for automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign#how-to)**. Once you complete those steps return to the user migration wizard and docs. 
```

> Some tips and recommendations for doing this step: 

Reviewer: I think you can just say tips or recommendations. I like tips; more conversational. :)

> * If you're already using a New Relic app for either Okta, Azure, or OneLogin, you're likely using an out-of-date one. The out-of-date apps are titled "New Relic by account" while the newer apps are titled "New Relic by organization."

Reviewer: ```suggestion
* If you're already using a New Relic app for either Okta, Azure, or OneLogin, you're likely using an out-of-date version. The out-of-date apps are titled "New Relic by account" while the newer apps are titled "New Relic by organization."
```

Reviewer: Or, well. I suggested "version," but any other word that fits should be fine. I think "one" is a little vague and there might exist a stronger noun somewhere out there.

> * Once you complete those steps, new user records are created on the new user model and synced in New Relic based on your identity provider configuration. In a later step (step #6), you'll migrate your users' assets to these new user records. 

Reviewer: ```suggestion
* Once you complete those steps, new user records are created on the new user model and synced in New Relic based on your identity provider configuration. Later in step #6, you'll migrate your users' assets to these new user records. 
```

> * After you complete provisioning users, you can confirm that these users have been added by seeing if they’re in the new [**User management** UI](#manage-users).

Reviewer: ```suggestion
* After you complete provisioning users, you can confirm that these users have been added by viewing their accounts in the new [**User management** UI](#manage-users).
```

* To access the new New Relic user management UI, you'll have to ensure you're logged in via your new user record: this may require logging out, logging back in, and verifying your email to see all logins associated with that email. 

> * For the step of granting access to groups in New Relic: for some basics of how access grants work, see [Access grants](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#understand-concepts). For a tutorial on how to create access grants, see the [user management tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model/#group-access).

Reviewer: A little unclear what the second bullet point is trying to convey. I think it should be fine to cut "For the step of granting access" and just include "For some basics about how access grants work"?

Alternatively, you can format it like this:

* To manage your granting access:
      - See our Access grants doc for basics
      - See our user management tutorial to create access grants

Or something similar.

* Note that you won't be able to edit users or groups in New Relic: all user and group changes must be done from your identity provider. 

When done, you can skip ahead to Step #5. 

### Login methods [#login-methods]

If you chose **Manually** for [managing users](#managing-users), you have a choice for how those users log in. You can select either a) email/password login or b) single sign on (SSO). Note that SSO is available only for organizations with Pro or Enterprise edition.

You can select multiple authentication methods based on your needs. For example, you might use SSO for your employees and username/password for contractors. 

If you select SSO, you must complete [additional configuration steps](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#saml). Some tips for doing this: 

> * If you're already using a New Relic app for either Okta, Azure, or OneLogin, you're likely using an out-of-date one. The out-of-date apps are titled "New Relic by account" while the newer apps are titled "New Relic by organization."

Reviewer: Same suggestion as above. Version works, or any other stronger noun to replace "one"

> * To access the new New Relic user management UI, you'll have to ensure you're logged in via your new user record: this may require logging out, logging back in, and verifying your email to see all logins associated with that email. 

Reviewer: I'd break this compound sentence into two separate ones. GH isn't showing me that handy edit tool, so I'll just type it out here:

* To access the new New Relic user management UI, you'll have to ensure you're logged in via your new user record. This may require logging out, logging back in, and verifying your email to see all logins associated with that email.
